packages/components/tooltip/src/Tooltip.tsx

```diff
@@ -8,30 +8,28 @@ import React, {
 } from 'react';
 import { usePopper } from 'react-popper';
 import { Placement } from '@popperjs/core';
-import cn from 'classnames';
+import { cx } from 'emotion';
 import type * as CSS from 'csstype';
 
-import styles from './Tooltip.css';
+import { styles } from './Tooltip.styles';
 
 import tokens from '@contentful/f36-tokens';
 
 import { Portal } from '@contentful/f36-utils';
+import type { CommonProps } from '@contentful/f36-core';
+import { Box, useId } from '@contentful/f36-core';
 
 export type TooltipPlace = Placement;
```

Comment:
Maybe this should be `TooltipPlacement` to keep it similar to the prop?

Reply:
Agree with Kristoffer